components/AdminItem/AdminItem.module.css

```diff
@@ -0,0 +1,130 @@
+.list-item {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  height: 68px;
+  padding-left: 2%;
+  padding-right: 2%;
+  border-top: 1px solid #EBEBEB;
+  color: #646464;
```

Comment:
for all the hard-coded hex codes you're using, if they're not defined within global.css, please add them there instead. This is so you can reuse these colors as `var(--color)` instead of hard coding. Also, if you decide to change this color, you'll only need to change it in the global.css file.